bulk_writer.js

```diff
@@ -140,7 +140,11 @@ BulkWriter.prototype.checkEsConnection = function checkEsConnection() {
   return new Promise((fulfill, reject) => {
     operation.attempt((currentAttempt) => {
       debug('checking for connection');
-      thiz.client.ping().then(
```

Comment:
Ping leads to "no active shards" error at bulk write while elasticsearch restarts.